shared/src/main/scala/coop/rchain/lmdb/LMDBStore.scala

```diff
@@ -37,10 +37,16 @@ final case class LMDBStore[F[_]: Sync](env: Env[ByteBuffer], dbi: Dbi[ByteBuffer
       withTxn(env.txnWrite)(f)
     }
 
-  def get(key: ByteBuffer): F[Option[ByteBuffer]] =
+  def get(key: ByteBuffer): F[Option[BitVector]] =
```

Comment:
Just a question on this. 
Is it reasonable to abstract this `BitVector` to `T`? Or that's too much

Reply:
I wanted to make minimal change on this file because it will be removed after RSpace will be moved to `KeyValueStore`.